examples/medplum-patient-intake-demo/src/App.tsx

```diff
@@ -26,6 +27,10 @@ export function App(): JSX.Element | null {
           title: 'Charts',
           links: [{ icon: <IconUser />, label: 'Patients', href: '/Patient' }],
         },
+        {
+          title: 'Management',
+          links: [{ icon: <IconFilePencil />, label: 'Customize intake form', href: '/customize-form' }],
```

Comment:
If you update the route as a mentioned below, then this link would be `Questionnaire/:id-of-intake-form/edit`